.azure/templates/system_test_general.yaml

```diff
@@ -30,20 +30,21 @@ jobs:
     - script: echo "##vso[task.setvariable variable=docker_tag]$([[ $(cat release.version) == *SNAPSHOT* ]] && echo "latest" || cat release.version )"
       displayName: "Set release tag as DOCKER_TAG for tests"
 
-    - bash: ".azure/scripts/setup_upgrade.sh"
-      displayName: "Setup environment for upgrade"
-      condition: eq('${{ parameters.groups }}', 'upgrade')
-
     - template: "general_steps.yaml"
 
     - bash: |
         make docker_build
         make docker_tag
+        docker images
```

Comment:
leftover?

Reply:
Yes, but I was thinking that this is "good to have" there. Gonna remove it, thanks.